### PR TITLE
applying hotfix

### PR DESCRIPTION
**Pull request by juliansmit-hpe**
## Summary and Scope
Add override to cray tftp container.

_Summarize what has changed. Explain why this PR is necessary. What is impacted? Is this a new feature, critical bug fix, etc?_

_Is this change backwards incompatible, backwards compatible, or a backwards compatible bugfix?_
Can helm rollback.

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [issue id](issue link) https://jira-pro.its.hpecorp.net:8443/browse/CASMCMS-8146

## Testing

_List the environments in which these changes were tested._

### Tested on:
Mug system.

### Test description:
Tested helm deployment on mug.

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)? Not sure
- Were continuous integration tests run? If not, why? No
- Was upgrade tested? If not, why? Yes
- Was downgrade tested? If not, why? Yes
- Were new tests (or test issues/Jiras) created for this change? No

## Risks and Mitigations

_Are there known issues with these changes? Any other special considerations?_

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: kubernetes/cray-tftpd-ipxe/values.yaml [hunk 1]
```diff
@@ -53,6 +53,9 @@ cray-service:
   replicaCount: 3
   containers:
     - name: cray-tftp
+      securityContext:
```
Comment (dlaine-hpe) [inline kubernetes/cray-tftpd-ipxe/values.yaml:56]: Shouldn't this be:
`securityContext:
    runAsUser: 0
     runAsGroup: 0
     runAsNonRoot: false
     readOnlyRootFilesystem: false
`

Comment (dlaine-hpe) [reply kubernetes/cray-tftpd-ipxe/values.yaml:56]: Can't get formatting to work nicely, but you get the point with the added values...

Comment (juliansmit-hpe) [reply kubernetes/cray-tftpd-ipxe/values.yaml:56]: I can make this change no problem. I just went ahead and applied the hotfix that went into Mercury into this PR.

File: kubernetes/cray-tftpd-ipxe/values.yaml [hunk 2]
```diff
@@ -53,6 +53,11 @@ cray-service:
...
+        runAsUser: 0
```
Comment (dlaine-hpe) [inline kubernetes/cray-tftpd-ipxe/values.yaml:57]: Does this need to run as root?  Default should not be, so adding this escalates the security privileges?

Comment (juliansmit-hpe) [reply kubernetes/cray-tftpd-ipxe/values.yaml:57]: I followed what was outlined in this ticket here https://jira-pro.its.hpecorp.net:8443/browse/CASMCMS-7942. 
It seems that without root this container fails with a permissions issue.

**Review by jsl-hpe — DISMISSED** [review]

**Review by jsl-hpe — DISMISSED** [review]

**Review by jsl-hpe — APPROVED** [review]

File: kubernetes/cray-tftpd-ipxe/values.yaml
Comment (mharding-hpe) [inline kubernetes/cray-tftpd-ipxe/values.yaml:1]: Copyright header in this file should be updated to include current year.

**Pull request closed without merging**